tests/fixtures/dos_multiple_briefs_fixture.json

```diff
@@ -44,6 +45,7 @@
       "additionalTerms":"You'll never work in this town again.",
       "applicationsClosedAt":"2016-03-22T00:00:00.000000Z",
       "backgroundInformation":"We'd like to build a thing to do athing.",
+      "specialistRole":"Content Specialist 2",
```

Comment:
Would be nice to have this as one of our real specialist roles e.g.
`"specialistRole": "deliveryManager"`

Note here that data coming back from the api sends the role type as camel case. However in our frontend we are going to need to show the user the pretty string of `Delivery manager` like we do on the page to view a brief it self:

![image](https://cloud.githubusercontent.com/assets/7228605/26691488/300b4014-46f5-11e7-902f-36e1765a9ce4.png)

This converting to a pretty string on the brief page itself might be done via the content loader but I might need to have a little look tomorrow to figure out how...